### DISCUSSION TITLE
Automatically clear a list of previously pasted (Send) files/text

### DISCUSSION THREAD

**Discussion by qupear** [localsend/localsend#1537] [Discussion (English, International)] [answered]
Hi. One question which maybe should be a feature request.
After I send a file it keeps staying in a list. I don't get the reason why this is default behavior. If I want to send another file I need to delete previous one. If not delete previous then it is duplicated on receiving sides.

I think it should be either automatically cleared successfully uploaded files or checked for duplicates on receiving sides. That way using app will be much smoother.

**Comment by Tienisto** [comment, marked as answer]
You need to select "Single Recipient" mode on the send tab
